README.md

```diff
@@ -26,6 +26,9 @@
 ```
 - Then open index.html in a live server.
 
+## Online version
+https://daf98.github.io/portfolio/
```

Comment:
- [ ] [OPTIONAL] This is not the right syntax for adding a live demo link because it doesn't give the user a clickable link. I kindly suggest you do it like below

`[Live-Demo](https://daf98.github.io/portfolio/)`

Reply:
Thank you!

Reply:
you're welcome